test/Dialect/ESI/services.mlir

```diff
@@ -157,18 +157,18 @@ msft.module @LoopbackCosimTop {} (%clk: i1, %rst: i1) {
 // CONN:         %MemA = sv.reg  : !hw.inout<uarray<20xi64>>
 // CONN:         %chanOutput, %ready = esi.wrap.vr %0, %write_done : none
 // CONN:         %rawOutput, %valid = esi.unwrap.vr %write, %ready : !hw.struct<address: i5, data: i64>
-// CONN:         %1 = hw.struct_extract %rawOutput["address"] : !hw.struct<address: i5, data: i64>
-// CONN:         %2 = hw.struct_extract %rawOutput["data"] : !hw.struct<address: i5, data: i64>
-// CONN:         %3 = comb.and %valid, %ready {sv.namehint = "write_go"} : i1
-// CONN:         %write_done = seq.compreg sym @write_done %3, %clk, %rst, %false  : i1
-// CONN:         %chanOutput_0, %ready_1 = esi.wrap.vr %5, %valid_3 : i64
+// CONN:         %address = hw.struct_extract %rawOutput["address"] : !hw.struct<address: i5, data: i64>
```

Comment:
I hate it when people are lazy in writing their tests!

Reply:
yea, right? 🤔